crates/derive/src/attributes/mod.rs

```diff
@@ -97,7 +98,11 @@ where
                     .await
                     .map_err(|e| PipelineError::BadEncoding(e).crit())?;
             sys_config
-                .update_with_receipts(&receipts, &self.rollup_cfg, header.timestamp)
+                .update_with_receipts(
+                    &receipts,
+                    self.rollup_cfg.l1_system_config_address,
+                    self.rollup_cfg.is_ecotone_active(header.timestamp),
```

Comment:
I'm not a huge fan of this API - we should just pass the rollup cfg ya? Op-alloy has it